aries-site/src/pages/learn/grid-fundamentals-part-one.mdx

```diff
@@ -0,0 +1,198 @@
+import { Notification, Paragraph } from 'grommet';
+import {
+  StepOne,
+  StepTwo,
+  StepThree,
+  StepFour,
+} from '../../examples/learn/tutorials/grid-fundamentals-part-1';
+
+In this tutorial you will use Grid to display a collection of products, similar to
+that of an e-commerce website, and use its [properties](https://v2.grommet.io/grid?theme=hpe#props)
+to:
+
+- Present a collection of items in a grid.
+- Allow the number of grid columns to adapt to a variety of screen widths.
```

Comment:
Wondering if there's a way in one of these bullets to include the term "responsive"

Reply:
done